### PR TITLE
Both File and Databroker browsers open at load; saved defaults

### PR DESCRIPTION
**Pull request by ronpandolfi**

### CODE REVIEW

File: xicam/gui/bluesky/search.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def __init__(self, catalog):
         self.new_results_catalog.connect(self.start_show_results)
 
         class ReloadThread(QThread):
-            def run(self):
+            def run(thread):
```
Comment (dylanmcreynolds) [inline xicam/gui/bluesky/search.py]: This comes up as as warning in in my IDE...should be self.

File: xicam/gui/bluesky/search.py [hunk 2]
```diff
@@ -130,7 +130,7 @@ def run(self):
         self.reload_thread.start()
 
         class ProcessQueriesThread(QThread):
-            def run(self):
+            def run(thread):
```
Comment (dylanmcreynolds) [inline xicam/gui/bluesky/search.py]: This comes up as as warning in in my IDE...should be self.

**Review by dylanmcreynolds — APPROVED** [review]
Looks good except I think the changes from 'self' to 'thread' aren't quite right. It doesn't really matter since the variables aren't used, but it is a python warning (at least in my IDE.)

File: xicam/gui/bluesky/search.py (hunk 1 above)
Comment (ronpandolfi) [reply xicam/gui/bluesky/search.py]: Moved these classes outside

**Pull request merged**